src/main/java/gregtech/api/recipes/RecipeBuilder.java

```diff
@@ -72,7 +73,7 @@ public RecipeBuilder(Recipe recipe, RecipeMap<R> recipeMap) {
         this.outputs.addAll(GTUtility.copyStackList(recipe.getOutputs()));
         this.chancedOutputs = new ArrayList<>(recipe.getChancedOutputs());
 
-        this.fluidInputs = GTUtility.copyFluidList(recipe.getFluidInputs());
+        this.fluidInputs = new ArrayList<>(recipe.getFluidInputs());
```

Comment:
So this change removes the fact that the inputs were copied. Do we want to have something that returns List<GTRecipeInput>, but copies the passed values?

Reply:
fluid inputs list is now a GTRecipeInput list, and they should be "immutable"